### DISCUSSION TITLE
无线信号丢失+无限重启

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#162] [Bugs/问题反馈]
最新源码本地编译AX6固件，从UBOOT刷入可正常启动，但修改WIFI参数后无线信号丢失，断电再启动开始无限重启无法进入系统......

**Comment by Boos4721** [comment]
any logs?

**Reply by ghost** [reply]
Sorry,none...无限重启我就直接刷回老版本了
